components/Footer.js

```diff
@@ -1,40 +1,86 @@
 import Link from 'next/link';
 import React from 'react';
+import styles from '../styles/components/Footer.module.scss';
 import SocialMedia from './SocialMedia';
 
+// TODO: add GM?
```

Comment:
works for me!

Reply:
Cool, we can do it once GM is reenabled.